nixos/modules/services/desktops/cpupower-gui.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cpupower-gui;
+in {
+  options = {
+    services.cpupower-gui = {
+      enable = mkEnableOption "cpupower-gui";
```

please add description
```suggestion
      enable = mkEnableOption "cpupower-gui";
      description = ''
        ...
      '';
```